rotary/utils.py

```diff
@@ -100,3 +100,39 @@ def symlink_or_compress(in_file_path, out_file_path):
         os.symlink(in_file_path, out_file_path)
     else:
         gzip_file(in_file_path, out_file_path)
+
+
+def get_contamination_reference_files(config_entry: list, db_path: str, phix_accession: str, human_accession: str):
+    """
+    Returns a list of file paths for contamination references based on short IDs
+
+    :param config_entry: value of config.get('contamination_references'); should be a list
+    :param db_path: path to the rotary database folder
+    :param phix_accession: NCBI assembly accession code for the PhiX genome assembly
+    :param human_accession: NCBI assembly accession code for the human genome assembly
+    :return: list of file paths for the contamination reference sequences
+    """
+
+    contamination_reference_paths = []
+
+    if isinstance(config_entry, list) is True:
+
```

Comment:
@jmtsuji I haven't brought this up before, but in some situations, you can drop the blank lines below the for and if statements and let the indentation handle the readability.

```python
if isinstance(config_entry, list) is True:
        for contamination_entry in config_entry:
            if contamination_entry.lower() == 'phix':
                contamination_reference_paths.append(os.path.join(db_path, 'contamination_references',
                                                                  f'PhiX__{phix_accession}.fna.gz'))
            elif contamination_entry.lower() == 'human':
                contamination_reference_paths.append(os.path.join(db_path, 'contamination_references',
                                                                  f'human__{human_accession}.fna.gz'))
            else:
                raise ValueError(f'Contaminant genome must be PhiX or human, but you provided {contamination_entry}')
else:
    raise TypeError(f'Contaminant genome names must be in a list, but you provided type {type(config_entry)}. '
                    'Try editing the config file and make sure that contamination_references is a list like '
                    '[PhiX] or [PhiX, human], not just "PhiX" or "human".')
```

Reply:
I usually use blank lines to break out logical blocks. For example:

```python
if x: # no space here because it's redundant because of the indent.
   data=some_func()
           # space here to break out the for loop away from the other code. Since blam is filled
   blam=[] # by the loop, we put it directly above the loop with no space and break it out.  
   for datum in data:
       if datum.cool == True: # Some libraries like numpy they have their own boolean values  
          blam.append(datum)  # (```np.bool```) so you have to use == to force an evaluation. 

   if len(blam) > 1: # Blank line between for loop and if statement since they are different logics.
       print(blam)
```

Reply:
@jmtsuji Just some examples of how I use spacing.